tests/test_tasking.py

```diff
@@ -24,3 +27,37 @@ def test_construct_order_parameters(tasking_mock):
     assert isinstance(order_parameters, dict)
     assert list(order_parameters.keys()) == ["dataProduct", "params"]
     assert order_parameters["params"]["acquisitionMode"] is None
+
+
+def test_get_quotations(tasking_mock):
+    get_quotations = tasking_mock.get_quotations()
+    assert len(get_quotations) == 23
+    workspace_id_filter = "80357ed6-9fa2-403c-9af0-65e4955d4816"
+    get_quotations = tasking_mock.get_quotations(workspace_id=workspace_id_filter)
+    quotations_accepted = (
+        quotation["workspaceId"] == workspace_id_filter for quotation in get_quotations
+    )
+    assert all(quotations_accepted)
+    get_quotations = tasking_mock.get_quotations(decision="ACCEPTED")
```

Comment:
perhaps test whether passing multiple `decision` in parameter work (since it accepts array).
And also add a test for `orderId` filter, i think those are the common filters that will be used